cor-journey-test-data/src/main/scala/tps/testdata/TdAll.scala

```diff
@@ -62,4 +62,27 @@ trait TdAll
       .foldLeft[List[(Journey, JourneyJson)]](Nil)(_ ++ _)
       .++(TdJourneyMultiPayment.allJourneys)
 
+  // only journeys that have a frontend journey within tps-frontend
+  @SuppressWarnings(Array(
+    "org.wartremover.warts.JavaSerializable",
+    "org.wartremover.warts.Serializable",
+    "org.wartremover.warts.Product"))
+  lazy val allTdFrontendJourneyInStates: List[TdJourneyInStates] = List(
```

Comment:
this was to make writing unit tests a bit easier in frontend (we don't have ui for MIB and PNGR)